Cargo.toml

```diff
@@ -19,13 +19,13 @@ serde_json = "1"
 erased-serde = "0.3"
 serde_derive = "1"
 futures = "0.3"
-hyper = "0.13"
-hyper-tls = "0.4"
+hyper = { version = "0.14", features = ["http1", "http2", "stream"] }
```

Comment:
Do we really need both: `http1` and `http2` support?

Reply:
You are right. http2 is not needed.